### ISSUE TITLE
unable to move or rename some items in MEGA

### ISSUE THREAD

**Issue by freebiesoft** [meganz/MEGAsync#1006]
![image](https://github.com/user-attachments/assets/2e48bb54-82df-40dc-a962-84a517f33c6f)

It doesn't even say which items? It's just ... blank?

**Comment by luisvalenzuelar** [comment]
I have this problem in Linux. It doesn't go away. So annoying. It happens when compiling latex. 

![image](https://github.com/user-attachments/assets/28e21730-2730-4ec7-95db-b006c234e74b)

**Comment by luisvalenzuelar** [comment]
Can this happen because the two computers do not have the same megasync version? To solve it, I had to select the cloud option, then go to sync debris, and manually restore from the cloud the edited documents.

**Comment by awvalenti** [comment]
Same happened to me today. Never had seen this problem before. I just made some changes on another PC (renamed some files and moved others to an ignored folder) and, when I got to this one, that window showed up. I tried selecting "In MEGA" as the "version I wanted to keep", but it didn't work. It kept loading forever. I tried restarting the app, restarting Linux, without success. Then I finally gave up and selected "Local" as the "version I wanted to keep". This made MEGA think it should upload some old stuff back into the cloud. I stopped it and erased the folders containing that stuff. Now MEGA cloud got totally crazy and told me I ran out of space, which I didn't. My local MEGA folder has normal size, but cloud says (on Recent Activity) that I re-uploaded more than 600 files. I can't understand what happened.
